eras/conway/test-suite/src/Test/Cardano/Ledger/Conway/Translation/GenerateGoldenFileMain.hs

```diff
@@ -0,0 +1,77 @@
+{-# LANGUAGE AllowAmbiguousTypes #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE FlexibleInstances #-}
+{-# LANGUAGE LambdaCase #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module Main where
+
+-- import Cardano.Ledger.Babbage.Core
+
+import Cardano.Ledger.Binary (mkSized)
+import Cardano.Ledger.Conway (Conway, ConwayEra)
+import Cardano.Ledger.Conway.TxBody (ConwayTxBody (..))
+import Cardano.Ledger.Core
+import Cardano.Ledger.Crypto
+import Cardano.Ledger.Language (Language (..))
+import Cardano.Ledger.TxIn (TxIn (..))
+import Data.Sequence.Strict (fromList)
+import qualified Data.Set as Set
+import Test.Cardano.Ledger.Alonzo.Translation.Golden (generateGoldenFile)
+import Test.Cardano.Ledger.Alonzo.Translation.TranslationInstanceGen (TranslatableGen (..))
+import qualified Test.Cardano.Ledger.Babbage.Translation.TranslatableGen as BabbageTranslatableGen (
+  genTx,
+  genTxOut,
+  utxoWithTx,
+ )
+import Test.Cardano.Ledger.Conway.Arbitrary ()
+import Test.QuickCheck (
+  Gen,
+  arbitrary,
+  listOf1,
+  scale,
+ )
+
+-- | Generates golden translation file for Conway era
+main :: IO ()
+main = generateGoldenFile @Conway [PlutusV1, PlutusV2, PlutusV3] "eras/conway/test-suite/golden/translations.cbor"
+
+instance TranslatableGen Conway where
```

Comment:
Let's keep the `main` file minimal and move the instance into `TranslatableGen` module

It will be easier to find them in the future.